.storybook/main.js

```diff
@@ -1,10 +1,16 @@
+// .storybook/main.js
+const {
+  getCodeEditorStaticDirs,
+} = require("storybook-addon-code-editor/getStaticDirs");
```

Comment:
Must be in package.json, `yarn add -dev storybook-addon-code-editor`

Reply:
It is not necessary. I forgot to remove it in a previous commit.